sdk/src/config/TESTNET.ts

```diff
@@ -335,6 +379,10 @@ const TESTNET_CONFIG: WormholeConfig = {
     evmos: 'https://evmos-testnet-rpc.polkachu.com',
     kujira: 'https://kujira-testnet-rpc.polkachu.com',
     klaytn: 'https://rpc.ankr.com/klaytn_testnet',
+    sepolia: 'https://rpc.ankr.com/eth_sepolia',
```

Comment:
only thing I notice is in the new SDK we're using a different RPC endpoint, but both are probably fine.

https://github.com/wormhole-foundation/connect-sdk/blob/main/core/base/src/constants/rpc.ts#L56